package/gluon-mesh-vpn-fastd/luasrc/lib/gluon/upgrade/400-mesh-vpn-fastd

```diff
@@ -15,11 +15,9 @@ users.remove_user('gluon-fastd')
 users.add_group('gluon-fastd', 800)
 
 
-local enabled = uci:get('fastd', 'mesh_vpn', 'enabled')
-if not enabled then
-  enabled = site.fastd_mesh_vpn.enabled and 1 or 0
-end
+local enabled = uci:get('fastd', 'mesh_vpn', 'enabled') and 1 or 0
```

Comment:
Please don't change unrelated code - in this case, you've even broken it by dropping `site.fastd_mesh_vpn.enabled`, which is what the whole `and 1 or 0` is about.